Uniswap/interface
# fix: init pool behavior

## Description


Improve new pool creation experience by:
1. Disabling increment / decrement buttons when min and max prices are undefined (i.e. when buttons have no effect)
2. Adding "Full Range" button, which allows the user to spread liquidity evenly across the entire curve in a V2 style


_Linear ticket:_ https://linear.app/uniswap/issue/WEB-2444/disable-plus-and-minus-buttons-when-starting-a-liquidity-pool


## Screen capture

### Before
|    Mobile    |   Desktop    |
| ------------ | ------------ |
| <img width="1512" alt="Mobile Before" src="https://github.com/Uniswap/interface/assets/18626088/0d75bf0e-00b2-401b-adc8-1062e834d515"> | <img width="1512" alt="Desktop Before" src="https://github.com/Uniswap/interface/assets/18626088/607156d4-7578-42f2-90f3-565aa0c61faf"> |


### After
|    Mobile    |   Desktop   |
| ------------ | ----------- |
| <img width="1512" alt="Mobile After" src="https://github.com/Uniswap/interface/assets/18626088/df6a62e3-8c7c-44ba-bf32-e8e0bccbe8e5">  | <img width="1512" alt="Desktop After" src="https://github.com/Uniswap/interface/assets/18626088/a4a55d7a-0e75-4947-b493-89c9268bc69e"> |

## Test plan

### QA (ie manual testing)
1. Visit the Add Liquidity page (V3)
2. Select a token that has at least one uninitialized fee tier
3. With this change applied, the increment / decrement buttons should be disabled until the user inputs starting values for the min / max prices
4. The "Full Range" button should be present and enabled; when clicked, it populates the min price input with zero and the max price input with infinity
